deploy/charts/rook-ceph/templates/deployment.yaml

```diff
@@ -125,3 +126,10 @@ spec:
         emptyDir: {}
       - name: webhook-cert
         emptyDir: {}
+      securityContext:
```

Comment:
Should this securityContext be merged with the same section above on line 34? It looks duplicative for the same container.

Reply:
YAML doesn't make this easy to read, but these security contexts are different. One is a pod security context, whereas the other is a "standard" security context. They have slightly different definitions, which is why both are required. Ideally everything should be defined in the pod security context where possible in order to enforce the security context for all containers.

https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.20/#podsecuritycontext-v1-core

https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.20/#securitycontext-v1-core